### ISSUE TITLE
BLOCKER - The "hamburger" menu toggle is impossible to use with a keyboard

### ISSUE THREAD

**Issue by sabderemane** [django/djangoproject.com#1409]
> Blocker: Critical issue. Certain users cannot perceive or operate this, with no workaround available.

This makes it impossible for keyboard and screen reader users to access menu items and navigate the site in this way. (https://github.com/django/djangoproject.com/issues/1234, [18:18](https://www.youtube.com/watch?v=dAiMPTOMh1U&t=1098s), [SC 2.1.1 Keyboard](https://www.w3.org/TR/WCAG22/#keyboard)).
